addons/mail/static/src/discuss/call/common/call_settings.xml

```diff
@@ -2,112 +2,132 @@
+                            <div class="d-flex w-100 flex-grow-1 align-items-center">
+                                <input class="flex-grow-2 form-range" type="range" min="0" max="20" step="1" t-att-value="store.settings.edgeBlurAmount" t-on-change="onChangeEdgeBlurAmount"/>
+                                <span class="p-1 text-end o-discuss-DiscussCallSettings-width-text-percentage"><t t-out="Math.floor(store.settings.edgeBlurAmount * 5)"/>%</span>
                             </div>
                         </label>
                     </div>
-                </t>
-            </div>
-            <div t-if="env.debug" class="d-flex flex-column px-3 overflow-auto">
-                <div class="py-2 fw-bolder text-700 text-truncate text-uppercase">Technical Settings</div>
-                <div class="mb-3 d-flex align-items-center flex-wrap">
-                    <label class="d-flex align-items-center flex-wrap mw-100 cursor-pointer" title="Log RTC events" aria-label="Log RTC events">
-                        <input type="checkbox" aria-label="Log RTC events" title="Log RTC events" t-on-change="onChangeLogRtcCheckbox" t-att-checked="store.settings.logRtc ? 'checked' : ''" class="form-check-input"/>
-                        <span class="ms-2 text-truncate">Log RTC events</span>
-                    </label>
                 </div>
-                <div t-if="store.settings.logRtc" class="mb-3 d-flex align-items-center flex-wrap">
-                    <button class="btn btn-primary" t-att-disabled="rtc.state.logs.size === 0" t-on-click="onClickDownloadLogs">Download logs</button>
+            </t>
+            <t t-if="env.debug">
+                <hr class="o-discuss-separator my-2"/>
+                <h4>Technical Settings</h4>
+                <div class="d-flex flex-column my-1">
+                    <t t-call="discuss.CallSettings.textToggler">
+                        <t t-set="text">Log RTC events</t>
+                        <t t-set="value" t-value="store.settings.logRtc"/>
+                        <t t-set="onchange" t-value="onChangeLogRtc"/>
+                    </t>
                 </div>
+                <button t-if="store.settings.logRtc" class="btn btn-primary mt-1 mb-2 w-100" t-att-disabled="rtc.state.logs.size === 0" t-on-click="onClickDownloadLogs">Download logs</button>
+            </t>
+        </div>
+    </t>
+    <t t-name="discuss.CallSettings.textToggler">
+        <div class="d-flex flex-wrap align-items-center">
+            <label class="d-flex align-items-center flex-wrap mw-100 cursor-pointer" t-att-title="text" t-att-aria-label="text">
+                <span class="text-truncate text-wrap" t-out="text"></span>
+            </label>
+            <div class="flex-grow-1"/>
+            <div class="form-check form-switch">
+                <input class="form-check-input" type="checkbox" role="switch" t-on-change="onchange" t-att-checked="value ? 'checked' : ''" t-att-title="text" t-att-aria-label="text"/>
```

Comment:
There's `cursor-pointer` on label but it doesn't trigger a change. Only the explicit click on `<input>` works